Proton/Sources/EditorCommand/Commands/ListCommand.swift

```diff
@@ -71,7 +71,8 @@ public class ListCommand: EditorCommand {
     public func execute(on editor: EditorView) {
         var selectedRange = editor.selectedRange
         // Adjust to span entire line range if the selection starts in the middle of the line
-        if let currentLine = editor.contentLinesInRange(NSRange(location: selectedRange.location, length: 0)).first {
+        if let currentLine = editor.contentLinesInRange(NSRange(location: selectedRange.location, length: 0)).first,
+            currentLine.range.length > 0 {
```

Comment:
This fixes the core issue of list style disappearing when list item is created between two non-list paragraphs.